networkit/cpp/io/DibapGraphReader.cpp

```diff
@@ -5,17 +5,36 @@
  *      Author: Henning
  */
 
-#if !defined _WIN32 && !defined _WIN64 && !defined WIN32 && !defined WIN64
+// networkit-format
+
+#ifndef NETWORKIT_WINDOWS
 
-#include <networkit/io/DibapGraphReader.hpp>
 #include <networkit/auxiliary/Log.hpp>
+#include <networkit/io/DibapGraphReader.hpp>
 
+#include <cstdio>
+#include <netinet/in.h>
```

Comment:
Is this include necessary?

Reply:
I think it is necessary for the little/big endian computations iirc.